src/selectors/profile.js

```diff
@@ -450,3 +451,43 @@ export const getHiddenTrackCount: Selector<HiddenTrackCount> = createSelector(
     return { hidden, total };
   }
 );
+
+export const getRelevantPagesForActiveTab: Selector<
```

Comment:
Please add some documentation for this function. I think it would be good to include documentation of what makes a page relevant.